docker-compose.yaml

```diff
@@ -90,8 +89,8 @@ services:
   ssv-node-4-dev:
     << : *default-dev
     container_name: ssv-node-4-dev
-    ports:
-      - 40008:40008
+#    ports:
```

Comment:
is the commenting out by purpose?

Reply:
yes, need to figure it out. @lior-blox what do you say?